### ISSUE TITLE
Rancher setup tutorial (Docker, Rancher, Kubernetes, Minecraft)

### ISSUE THREAD

**Issue by hakova** [techno-tim/youtube-videos#11]
Dear Tim,

Thank you for the awesome video. I suspect that the instructions in the video may be outdated and not possible to reproduce anymore. I posted a comment on YouTube for that video with the name hakovatube, and I don't want to state the same thing here. Briefly, I tried 3 OSes in VMs (CentOS 8.2, Ubuntu 20.10 and Ubuntu 20.4), spent 2 nights just to be able to add a cluster with no avail. I tried publishing ports to 9090 and 9091 instead of keeping them on 80 and 443, which didn't have any positive impact, and in some cases, rancher became unreachable.The curl script didn't work for any of the above-mentioned OSes, I ended up installing docker from release repositories. I found out that docker from snap requires user's home folder to be the location of persistent settings and therefore should not be used. I may still be missing something but please feel free to start up a new VM and try to reproduce your steps in this video; I suspect that you would find that it is not reproducible anymore.

Thanks.

**Comment by timothystewart6** [comment]
Did you see the updated command here?  https://github.com/techno-tim/youtube-videos/tree/master/docker-rancher-kubernetes

**Comment by timothystewart6** [comment]
@hakova also, this command is directly from https://rancher.com/docs/rancher/v2.x/en/installation/other-installation-methods/single-node-docker/

**Comment by hakova** [comment]
Thank you @timothystewart6 for your quick replies. Yes, I saw the updated command and have been using it. I have also been trying to get this work since my first message without luck. A new cluster just cannot be created for whatever reason, and goes into an infinite loop of repeating messages such as "deploying port listener containers", "Running control plane -> worker port checks", "[network] Successfully started [rke-port-checker] container on host [192.168.1.136]", "[remove/rke-cp-port-listener] Successfully removed container on host [192.168.1.136]", and so on. I tried deploying workloads on the local cluster instead of creating a new cluster, some of which seemed like succeeded, but I could not publish their ports to the host for reasons beyond my understanding. It is really not too confusing to get the settings right, but somehow they don't seem to work, and the open port links on rancher UI always directs to the rancher docker's IP rather than host machine's. Am I supposed to publish rancher docker's ports to the host machine as well? That means restarting rancher docker every time a new workload is added, and that doesn't sound right to me. This is a separate issue though. I really think that not being able to create a new cluster, after following your guide, reading pages of documentation, etc. makes it unlikely that I am missing something so obvious.
I am working on a KVM/QEMU VM that now has Ubuntu 18.4.05 server. I am relatively comfortable in working with docker, so most concepts here are not totally new to me. I will try a little more but I have a life and family to tend to, I cannot obsess with this forever, lol.

**Comment by hakova** [comment]
Thank you @timothystewart6 for your quick replies. Yes, I saw the updated command and have been using it. I have also been trying to get this work since my first message without luck. A new cluster just cannot be created for whatever reason, and goes into an infinite loop of repeating messages such as "deploying port listener containers", "Running control plane -> worker port checks", "[network] Successfully started [rke-port-checker] container on host [192.168.1.136]", "[remove/rke-cp-port-listener] Successfully removed container on host [192.168.1.136]", and so on. I tried deploying workloads on the local cluster instead of creating a new cluster, some of which seemed like succeeded, but I could not publish their ports to the host for reasons beyond my understanding. It is really not too confusing to get the settings right, but somehow they don't seem to work, and the open port links on rancher UI always directs to the rancher docker's IP rather than host machine's. Am I supposed to publish rancher docker's ports to the host machine as well? That means restarting rancher docker every time a new workload is added, and that doesn't sound right to me. This is a separate issue though. I really think that not being able to create a new cluster, after following your guide, reading pages of documentation, etc. makes it unlikely that I am missing something so obvious.
I am working on a KVM/QEMU VM that now has Ubuntu 18.4.05 server. I am relatively comfortable in working with docker, so most concepts here are not totally new to me. I will try a little more but I have a life and family to tend to, I cannot obsess with this forever, lol.
